### PR TITLE
[COMSUP-337] Add links to homepage sections

### PR DESCRIPTION
**Pull request by tristanlee85**

### CODE REVIEW

**Comment by tristanlee85** [comment]
@jsernaque You mentioned about adding a section for API references. I am happy to add them, but I think we need to review the items we currently have to slim them down before adding more.

**Review by jsernaque — CHANGES_REQUESTED** [review]
Broken links for: Applications sub-headings and REST API. Fix the REST API link by setting it to a full URL. 
The API sub-heading requires a path. What if we renamed it to Develop and link to the Develop page?
Suggestion: We should align the Applications landing page with the main landing page. 
Suggestion: We should increase the font on the main landing page.

File: src/config/home/sections.ts [hunk 1]
```diff
@@ -151,11 +151,45 @@ export const sections: HomepageSectionGroup[] = [
...
           },
         ],
       },
+      {
+        title: 'Develop',
+        path: 'develop',
+        items: [
+          {
+            title: 'REST API',
+            path: '/rest_api/',
```
Comment (jsernaque) [inline src/config/home/sections.ts:185]: This link still returns a Page not found.

Comment (tristanlee85) [reply src/config/home/sections.ts:185]: Pushed a fix. I had verified this as working on my end but I think I had it cached.

**Review by jsernaque — APPROVED** [review]
LGTM

**Pull request merged**